integrations/qdrant/src/haystack_integrations/document_stores/qdrant/document_store.py

```diff
@@ -385,9 +492,8 @@ def _set_up_collection(
             # Create Payload index if payload_fields_to_index is provided
             self._create_payload_index(collection_name, payload_fields_to_index)
             return
-
-        current_distance = collection_info.config.params.vectors.distance
-        current_vector_size = collection_info.config.params.vectors.size
+        current_distance = collection_info.config.params.vectors["text-dense"].distance
```

Comment:
A constant for the vector names would be better since the literals have several occurrences.

Reply:
agree, constant for "dense embedding" vector name and for "sparse embedding" vector name. Not entierly sure where/how to put this in the class

Reply:
Replacing all occurrences of `"text-dense"` and `"text-dense"` literals in this file with a constant.